modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of your builds and runs, you can use custom security context constraints (SCCs).
+
+.Procedure
+
+Create a `pipelines-scc` Security Context Constraints (SCC) and configure its `runAsUser` and `seLinuxContext` fields.
++
```

Comment:
```suggestion

```
Remove `+` to fix `.Example` heading that doesn't render correctly

Reply:
Retained the +, but I hope it works now.